qoe-metrics-dashboard/src/app/pages/consumption/detail/consumptionDetailPage.tsx

```diff
@@ -0,0 +1,141 @@
+import { Box, CircularProgress, Table, TableBody, TableCell, TableHead, TableRow, Typography } from '@mui/material';
+import { useContext } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { EnvContext } from '../../../env.context';
+import { useConsumptionReportDetail } from '../../../hooks/consumption-api';
+import React from 'react';
+
+import './ConsumptionDetailPage.scss';
+
+function ConsumptionDetailPage() {
+    const envCtx = useContext(EnvContext);
+
+    const [searchParams] = useSearchParams();
+
+    const { reportDetails, error, loading } = useConsumptionReportDetail(envCtx.backendUrl, searchParams);
+
+    if (loading) {
+        return (
+            <div className="loading">
+                <CircularProgress />
+            </div>
+        );
+    }
+
+    if (error) {
+        return <div>{error}</div>;
+    }
+
+    if (!reportDetails) {
+        return <div>No Data found</div>;
+    }
+
+    return (
+        <Box
+            padding={'2rem'}
+            component={'div'}
+            bgcolor={'background.default'}
+            display={'flex'}
+            flexDirection={'column'}
+            alignItems={'center'}
+            height={'100%'}
+            sx={{
+                overflowY: 'scroll',
+                overflowX: 'hidden',
+            }}
+        >
+            <Box display={'flex'} flexDirection={'column'} gap={'2rem'} width={'90%'} maxWidth={900}>
+                <Box
+                    padding={'2rem'}
+                    bgcolor={'background.paper'}
+                    borderRadius={'2rem'}
+                    alignItems={'flex-start'}
+                    display={'flex'}
+                    flexDirection={'column'}
+                    className="box-shadow"
+                >
+                    <Typography
+                        component={'h2'}
+                        variant="h5"
+                        paddingBottom={'1rem'}
+                        fontFamily={'Roboto Mono'}
+                        color={'primary'}
+                    >
+                        Consumption Detail Report
+                    </Typography>
+                    <Box component={'div'} width={1} sx={{ overflowX: 'auto' }}>
+                        <Table>
+                            <TableBody>
+                                <TableRow>
+                                    <TableCell className="mono bold">Client ID</TableCell>
```

Comment:
Hier würde ich die Verwendung von mono ein bisschen reduzieren. Siehe Kommentar

Reply:
Das hattest du auch bei dem basic information report so gemacht 

<img width="906" alt="grafik" src="https://github.com/KonsumGandalf/awt-pj-ss24-5gmag-2/assets/103542043/10c1f02e-b821-4499-820d-6bd02df9a9fe">

Reply:
habe das jetzt schonmal zu camel case gemacht
<img width="1094" alt="grafik" src="https://github.com/KonsumGandalf/awt-pj-ss24-5gmag-2/assets/103542043/30abbc86-d9da-4e42-b642-e805743ace9b">

Reply:
Ja ich meinte auch nur, dass ich mono bei den Values wegnehmen würde, außer bei der Id. In der unteren Tabelle hättest du die Überschriften meinetwegen auch so lassen können. Da hätte ich mono jetzt noch bei den values weggenommen.

Reply:
Und ergibt es Sinn den Server Endpoint als link zu machen?

Reply:
good point, habe den link hinzugefügt und die mono rausgenommen:

<img width="1716" alt="grafik" src="https://github.com/KonsumGandalf/awt-pj-ss24-5gmag-2/assets/103542043/027eb597-4862-40d4-915b-a4cfa74cfdbb">

Reply:
der link zeigt jetzt auf

https://livesim.dashif.org/